docs/source/essentials/migration.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Migration Guides
```

Comment:
Nitpicking: call this file migrating-to-1-3-x.md ? We will most likely need migrating-to-2-0-x.md very soon :)

Reply:
I was thinking we can add 2.x also here. If we use a header it will also show up at the right nav bar.

Reply:
👍